### ISSUE TITLE
Problem with direct navigation to routes

### ISSUE THREAD

**Issue by gep13** [emberjs/ember.js#14913]
Apologies if this has been asked elsewhere before, but I couldn't find anything.

We are facing an issue with regard to direct navigation to URL's when we publish our ember application onto our Apache Server.  What I mean by this is...

If we go to the root of our application, it loads happily in the browser, i.e. if I navigate to here: http://10.0.100.188/embertemplate/

On that page, we have a button to go to the Login page, and if I click on that the URL in the address bar changes to this: http://10.0.100.188/embertemplate/login and the login template opens up.

However, if I take that last URL and paste it directly into the browser, I get the following:

![image](https://cloud.githubusercontent.com/assets/1271146/22702870/1473e40e-ed5a-11e6-9fcd-7e817eec089e.png)

When I do the above when running locally using `ember serve` the direct navigation to routes working exactly as expected, which leads me to believe that this is an Apache configuration issue.  Can anyone suggest what might be required to get this to work?

Thanks in advance!

**Comment by locks** [comment]
Hi @gep13, this is indeed a problem with your Apache configuration.
See https://guides.emberjs.com/v2.11.0/tutorial/deploying/#toc_apache for a suggested resolution to this problem. Alternatively you could set your application's `locationType`, consult a more detailed explanation here: https://guides.emberjs.com/v2.11.0/configuring-ember/specifying-url-type/ .

I'll close the issue since it isn't related to Ember. Reach us on Slack for further assistance!

**Comment by gep13** [comment]
@locks thank you for the very prompt response. I will give this a try tomorrow.  Thank you!

**Comment by gep13** [comment]
For anyone else hitting this thread, I was able to get this working eventually.  Using the configuration shown here:

![image](https://cloud.githubusercontent.com/assets/1271146/22730203/e300883a-eddd-11e6-9432-443a028cf9e0.png)

from this link (https://guides.emberjs.com/v2.11.0/tutorial/deploying/#toc_apache), was the first part of it, but after adding this, it still wasn't working.

I changed the permissions on the `.htaccess` file thinking that that would be it, but that didn't do anything (`chmod 755 .htaccess`).

I tried restarting apache, but that didn't help either (`sudo service httpd restart`).

Eventually, I stumbled onto this section of the `httpd.conf` file:

![image](https://cloud.githubusercontent.com/assets/1271146/22730257/2ae82c02-edde-11e6-8d6c-79628cbd8558.png)

This was set to None.  By switching to All, it all started working.
